game/scripts/vscripts/Sections/Chapter6/SectionClosing.ts

```diff
@@ -176,13 +188,67 @@ function onStart(complete: () => void) {
         waitNpcsSpawned(),
 
         // Main logic
-        tg.fork([
+        tg.forkAny([
             // Play dialog
             tg.seq([
                 tg.audioDialog(LocalizationKey.Script_6_Closing_1, LocalizationKey.Script_6_Closing_1, slacks),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_2, LocalizationKey.Script_6_Closing_2, sunsFan),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_3, LocalizationKey.Script_6_Closing_3, slacks),
                 tg.audioDialog(LocalizationKey.Script_6_Closing_4, LocalizationKey.Script_6_Closing_4, sunsFan),
+
+                tg.withHighlights(
+                    tg.forkAny([
+                        tg.seq([
+                            tg.immediate(_ => {
+                                goalDestroyTowers.start()
+                                goalDestroyTowers.setValue(0)
+                                pathParticleID = createPathParticle([...pathLocations, ancient.GetAbsOrigin()])
+                            }),
+                            tg.completeOnCheck(_ => {
+                                towersToDestroy = towersToDestroy.filter(tower => unitIsValidAndAlive(tower))
+                                goalDestroyTowers.setValue(2 - towersToDestroy.length)
+
+                                return towersToDestroy.length === 0
+                            }, 0.1)
+                        ]),
+
+                        tg.seq([
+                            tg.panCameraExponential(getPlayerCameraLocation(), ancient.GetAbsOrigin(), 2),
```

Comment:
This gets the player camera location when the graph is created, you want `_ => getPlayerCameraLocation()` here

Reply:
One day I'll remember this properly. One day. 

Fixed.